crates/ash_cli/src/utils/templating.rs

```diff
@@ -1156,6 +1156,30 @@ pub(crate) fn template_avalanche_subnet_props_table(
     props_table
 }
 
+pub(crate) fn template_blockscout_props_table(
+    blockscout: &console::api_models::GetAllProjectResources200ResponseInner,
+) -> Table {
+    let mut props_table = Table::new();
+    props_table.set_format(*format::consts::FORMAT_NO_BORDER_LINE_SEPARATOR);
+
+    props_table.add_row(row![
+        "IP address".bold(),
+        type_colorize(&blockscout.blockscout_ip.clone().unwrap_or_default()),
+    ]);
```

Comment:
This does not seem to work.

I'm getting:

```
+---------------+-------------+-----------------+---------------------+--------+------------------+---------+-------------------------------------------------------------------+
| blockscout    | 7053...982a | Blockscout      | azure/francecentral | Medium | 2024-06-06T15:07 | Running |  IP address |                                                     |
|               |             |                 |                     |        |                  |         |  Running    | true                                                |
+---------------+-------------+-----------------+---------------------+--------+------------------+---------+-------------------------------------------------------------------+
```

Whereas the `blockscoutIp` can be fetched successfully in --json mode:

```
cargo run -- console resource info blockscout --json | jq
   Compiling ash_sdk v0.4.3 (/home/leo/Ash/ash-rs/crates/ash_sdk)
   Compiling ash_cli v0.4.3 (/home/leo/Ash/ash-rs/crates/ash_cli)
    Finished dev [unoptimized + debuginfo] target(s) in 7.29s
     Running `target/debug/ash console resource info blockscout --json`
{
  "blockscoutConfig": {
    "blockscoutConfig": {}
  },
  "blockscoutIp": "20.19.244.45",
  "blockscoutStatus": {
    "restartRequired": false,
    "running": true
  },
  "cloudRegionId": "094fcf66-2cb7-416c-8e4e-5069eba0b8cd",
  "created": "2024-06-06T15:07:43.573921",
  "id": "70531311-3cb7-46ff-830a-d74824e3982a",
  "name": "blockscout",
  "ownerId": "9b80e516-5f06-4983-b955-0a5a146afa60",
  "projectId": "9af0e486-4571-4f45-816e-90140b9c3d43",
  "resourceType": "blockscout",
  "size": "medium",
  "status": "running"
}
```

Reply:
fixed in https://github.com/AshAvalanche/jeeo/pull/155/commits/e031e60af9829b82978075f1fe13e2fedbcf9efc